pkg/xen-tools/initrd/chroot2.c

```diff
@@ -10,6 +11,7 @@
 #include <sys/wait.h>
 #include <unistd.h>
 #include <pwd.h>
+#include <grp.h>
 
 typedef struct clone_args clone_args;
```

Comment:
Now that you change the declaration of _parsed_args_ in _child_func()_, you don't need this typedef anymore, isn't being in use in any place....